### DISCUSSION TITLE
Update failed

### DISCUSSION THREAD

**Discussion by vquie** [JulianPrieber/llc-docker#35] [Issues]
I cannot apply the most recent update. It says that the backup is created and exits with status 500 when calling `/update?finishing`. I tried with and without proxy.
After the error 500 maintenance mode is enabled. I found out I had to remove /htdocs/storage/MAINTENANCE to bail out of maintenance mode.

Are there any logs that may help?

**Comment by JulianPrieber** [comment]
Apologies for the late reply.

Logs mostly won't help me much. To know what's going on, I have to know the reason for the error 500.
An error 500 means that the server encountered an unexpected condition that prevented it from fulfilling the request. It's the most generic error you can get and can be caused by pretty much anything in the updater.
You may find an error description in your Apache error log.

If you can reproduce the error that this would help very much.
In your config if you have the debug mode enabled you will get a detailed stack trace on any error.
This would most likely tell us what exactly went wrong.

This error page has a built-in share function, if you can, please send a link to this error here.
